### DISCUSSION TITLE
drawSdJpeg(file.name(), 0, 0); Random file

### DISCUSSION THREAD

**Discussion by Pedrolasis** [Bodmer/TFT_eSPI#1938] [Q&A - General]
Hello,

Is it possible for the program in your example "ESP32_SDcard_jpeg" to display random jpg's on the sd card ?
Using numbers for the jpg's on the sd card rather than names ...........
drawSdJpeg("random(1,10)", 0, 0);   < I know this wont work but as example of what I am asking.

I have searched and searched, tried and tried, posted in forums, but no luck.

Many thanks.

**Comment by Bodmer** [comment]
Yes, the drawSdJpeg function expects a string, so you need to create a string with the number in it plus the .jpg extension. For example:
```
  char filename [20]; // 19 chars max + terminating null
  int i = 42;
  itoa(i,filename,10); // Put string version of a number in filename, number base 10
  strcat(filename, ".jpg"); // add .jpg to end of string
  drawSdJpeg(filename, 0, 0); 
```

**Reply by Pedrolasis** [reply]
Hello Bodmer,

Thank you so much for pointing it out.

Much appreciated :)
